.gitlab/build_lassen.yml

```diff
@@ -27,14 +27,14 @@
 .src_build_on_lassen:
   stage: build
   variables:
-    ALLOC_COMMAND: "lalloc 1 -W 25 -q pdebug"
+    ALLOC_COMMAND: "lalloc 1 -W 25 -q pdebug -alloc_flags atsdisable"
```

Comment:
I recall there were a few instances where we would randomly get an ATS-disabled node, and GPU tests would fail because of that (see PR #843). By disabling ATS explicitly, we should be able to at least get consistent failures instead of random failures.

In any case, I think being able to detect when we're accessing host memory from the device by loudly failing in CI should be helpful in resolving performance issues related to ATS; I think there was an issue with ImplicitGrid where we were unintentionally allocating the buffers in host memory, resulting in a drastic slowdown from ~100s of ms to seconds in the `mesh_tester` example.

Reply:
Thanks @publixsubfan -- I completely agree and it's great to fix those in our testing.